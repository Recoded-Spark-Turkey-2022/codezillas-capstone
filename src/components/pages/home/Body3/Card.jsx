import React from 'react';

function Card({ imgSource, altText, header, text }) {
  return (
    <div className=" flex items-stretch">
      <div className="w-full max-w-sm  bg-white border border-gray-200 rounded-3xl shadow-lg dark:border-gray-700 p-6 mb-5  md:shrink-0 items-center ">
        <div className=" flex flex-col justify-center items-center ">
          <img
            className="object-scale-down h-20 w-25 md:object-scale-down "
            src={imgSource}
            alt={altText}
          />
        </div>
        <h3 className="text-4xl font-medium pt-8 p-2 flex flex-col justify-center items-center">
          {header}
        </h3>
        <p className="py-2 pt-4 p-2 text-center flex flex-col justify-center items-center">
          {text}
        </p>
      </div>
    </div>
  );
}

export default Card;
