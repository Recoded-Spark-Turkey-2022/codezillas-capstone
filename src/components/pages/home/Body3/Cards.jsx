import React from 'react'
import chat from './images/chat.png'
import phone from './images/phone.png'
import video from './images/video.png'

const Cards = () => {
  return (
    <div className='w-full h-screen bg-white flex justify-center items-center'>
    <div className='w-8/12 h-5/6 border flex flex-col items-start justify-evenly '>
      <div className='   text-5xl font-medium uppercase flex flex-col md:flex  md:max-w-5xl items-start justify-evenly '><h3>we can communicate through</h3></div>
      <div className=' container  mx-auto flex flex-wrap items-center justify-center my-16 '>   
   <div className=' lg:w-1/4 w-full lg:pr-3 items-stretch flex-1 h-80  md:flex  mx-auto md:max-w-lg'>
    <div className=' w-full max-w-sm bg-white border border-gray-200 rounded-3xl shadow-lg dark:bg-gray-800 dark:border-gray-700 p-6  md:shrink-0 '>
        <div className=' flex flex-col justify-center items-center '>
    <img className="object-scale-down h-20 w-25 md:object-scale-down " src={phone} alt="phone" />
    </div>
        <h3 className="text-4xl font-medium pt-8 p-2 flex flex-col justify-center items-center">Voice Call</h3>
        <p className='py-2 pt-4 p-2 text-center flex flex-col justify-center items-center'>Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!</p>

        </div>
    </div>
    <div className=' lg:w-1/4 w-full lg:pr-3 items-stretch flex-1 h-80  md:flex  mx-auto md:max-w-lg'>
    <div className= 'w-full max-w-sm bg-white border border-gray-200 rounded-3xl shadow-lg dark:bg-gray-800 dark:border-gray-700 p-6   md:shrink-0 '>
    <div className=' flex flex-col justify-center items-center '>
    <img className="object-scale-down h-20 w-25 md:object-scale-down " src={chat} alt="chat" />
    </div>
        <h3 className="text-4xl font-medium pt-8 p-2 flex flex-col justify-center items-center">Chat</h3>
        <p className='py-2 pt-4 p-2 text-center flex flex-col justify-center items-center'>Need to talk to someone?Come have a chat with your therapist!</p>

        </div>
    </div>
    <div className=' lg:w-1/4 w-full lg:pr-3 items-stretch flex-1 h-80  md:flex  mx-auto md:max-w-lg '>
    <div className=' w-full max-w-sm bg-white border border-gray-200  rounded-3xl shadow-lg dark:bg-gray-800 dark:border-gray-700 p-6  md:shrink-0 '>
    <div className=' flex flex-col justify-center items-center '>
    <img className="object-scale-down h-20 w-25 md:object-scale-down " src={video} alt="video" />
    </div>
        <h3 className="text-4xl font-medium pt-8 p-2 flex flex-col justify-center items-center">Video Call</h3>
        <p className='py-2 pt-4 p-2 text-center flex flex-col justify-center items-center'>For better experience therapists recommend video calls, but always remember that its a choice!</p>

        </div>
    </div>
</div>  
</div>
</div> 
   
  )
}

export default Cards