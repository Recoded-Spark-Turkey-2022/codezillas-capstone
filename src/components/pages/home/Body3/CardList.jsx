import React from 'react';
import Card from './Card';
import chat from './images/chat.png';
import phone from './images/phone.png';
import video from './images/video.png';

const Cards = () => {
  return (
    <div className="justify-center  flex flex-wrap items-starts  ">
      <div className="flex items-center justify-center mb-20 ">
        <div className=" flex flex-wrap gap-5 justify-center mb-20 ">
          <Card
            imgSource={phone}
            altText="phone"
            header="Voice Call"
            text="Feeling ready to start a conversation? Give your therapist a voice
              call and talk your heart out!"
          />

          <Card
            imgSource={chat}
            altText="chat"
            header="Chat"
            text="Need to talk to someone?Come have a chat with your therapist!"
          />

          <Card
            imgSource={video}
            altText="video"
            header="Video Call"
            text="For better experience therapists recommend video calls, but
                always remember that its a choice!"
          />
        </div>
      </div>
    </div>
  );
};

export default Cards;
