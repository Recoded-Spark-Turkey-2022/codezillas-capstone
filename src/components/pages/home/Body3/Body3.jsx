import React from 'react'
import Cards from './Cards'

const Body3 = () => {
  return (
    <div>
      <div className=' bg-white flex  justify-center  max-w-[ 1440px] min-w-[500px] '>
    <div className='w-8/12 h-screen flex flex-col items-start  justify-evenly min-h-[1400px] '>
    <div className='  text-5xl font-medium uppercase font-sans mt-40 mb-10'><h3>we can communicate through</h3></div>
         
<Cards /> 


</div>
</div>
</div>

  
  )
}

export default Body3