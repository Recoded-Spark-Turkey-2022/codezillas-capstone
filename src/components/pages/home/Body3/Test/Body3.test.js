import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Body3 from '../Body3';
import { store } from '../../../../../app/store';

it(`renders correctly`, () => {
  const Body3comp = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <Body3 />
        </BrowserRouter>
      </Provider>
    )
    .toJSON();
  expect(Body3comp).toMatchSnapshot();
});
