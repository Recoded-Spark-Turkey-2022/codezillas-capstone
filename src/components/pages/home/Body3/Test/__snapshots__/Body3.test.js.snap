// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className=" bg-white flex  justify-center  max-w-[ 1440px] min-w-[500px] "
  >
    <div
      className="w-8/12 h-screen flex flex-col items-start  justify-evenly min-h-[1400px] "
    >
      <div
        className="  text-5xl font-medium uppercase font-sans mt-40 mb-10"
      >
        <h3>
          we can communicate through
        </h3>
      </div>
      <div
        className="   justify-center  flex flex-wrap items-starts  "
      >
        <div
          className="flex items-center justify-center mb-20 "
        >
          <div
            className=" flex flex-wrap gap-5 justify-center mb-20 "
          >
            <div
              className=" max-w-sm bg-white border border-gray-200 rounded-3xl shadow-lg  dark:border-gray-700 p-6 mb-5 "
            >
              <div
                className=" flex flex-col justify-center items-center "
              >
                <img
                  alt="phone"
                  className="object-scale-down h-20 w-25 md:object-scale-down "
                  src="phone.png"
                />
              </div>
              <h3
                className="text-4xl font-medium pt-8 p-2 flex flex-col justify-center items-center"
              >
                Voice Call
              </h3>
              <p
                className="py-2 pt-4 p-2 text-center flex flex-col justify-center items-center"
              >
                Feeling ready to start a conversation? Give your therapist a voice call and talk your heart out!
              </p>
            </div>
            <div
              className=" flex items-stretch"
            >
              <div
                className="w-full max-w-sm  bg-white border border-gray-200 rounded-3xl shadow-lg dark:border-gray-700 p-6 mb-5  md:shrink-0 items-center "
              >
                <div
                  className=" flex flex-col justify-center items-center "
                >
                  <img
                    alt="chat"
                    className="object-scale-down h-20 w-25 md:object-scale-down "
                    src="chat.png"
                  />
                </div>
                <h3
                  className="text-4xl font-medium pt-8 p-2 flex flex-col justify-center items-center"
                >
                  Chat
                </h3>
                <p
                  className="py-2 pt-4 p-2 text-center flex flex-col justify-center items-center"
                >
                  Need to talk to someone?Come have a chat with your therapist!
                </p>
              </div>
            </div>
            <div
              className=" flex items-stretch "
            >
              <div
                className=" w-full max-w-sm bg-white border border-gray-200  rounded-3xl shadow-lg  dark:border-gray-700 p-6 mb-5 md:shrink-0 "
              >
                <div
                  className=" flex flex-col justify-center items-center "
                >
                  <img
                    alt="video"
                    className="object-scale-down h-20 w-25 md:object-scale-down "
                    src="video.png"
                  />
                </div>
                <h3
                  className="text-4xl font-medium pt-8 p-2 flex flex-col justify-center items-center"
                >
                  Video Call
                </h3>
                <p
                  className="py-2 pt-4 p-2 text-center flex flex-col justify-center items-center"
                >
                  For better experience therapists recommend video calls, but always remember that its a choice!
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
